.github/pull_request_template.md

```diff
@@ -32,3 +32,4 @@
 - [ ] I have not added any markdown to labels, titles and button text in config
 - [ ] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
 - [ ] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`
+- [ ] I have updated any relevant `.env` values in both `.env-template` and in Bitwarden
```

Comment:
[dust] We can drop `any` from `I have updated any relevant `.env` values in both `.env-template` and in Bitwarden`